docs/src/tutorials/magnetism.ipynb

```diff
@@ -0,0 +1,577 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29d5d62d-af4a-416d-bbe2-1338d32b30f5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import scipp as sc\n",
+    "\n",
+    "import easyreflectometry\n",
+    "import refl1d\n",
+    "import refl1d.names\n",
+    "\n",
+    "from easyreflectometry.calculators import CalculatorFactory\n",
+    "from easyreflectometry.experiment import Model\n",
+    "from easyreflectometry.experiment import PercentageFhwm\n",
+    "from easyreflectometry.sample import Layer\n",
+    "from easyreflectometry.sample import Material\n",
+    "from easyreflectometry.sample import Multilayer\n",
+    "from easyreflectometry.sample import Sample"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fd3e8f7-84ac-41c4-a89d-922ed82a001e",
+   "metadata": {},
+   "source": [
+    "For reference we fetch the version of the software packages we are using. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "549734c1-bbd9-41f3-8a20-d7a8ded37802",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'numpy: {np.__version__}')\n",
+    "print(f'scipp: {sc.__version__}')\n",
+    "print(f'easyreflectometry: {easyreflectometry.__version__}')\n",
+    "print(f'refl1d: {refl1d.__version__}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dad78ccc-1e6f-47cd-8557-c4fa6c736f4b",
+   "metadata": {},
+   "source": [
+    "## Building our model\n",
+    "\n",
+    "The system that was used to produce the data shown above is based on a silicon subphase with two layers upon it. \n",
+    "These two layers are charachterized by having a scattering length density (SLD) of respectively 4 and 8.\n",
+    "Both layers have a rougness of 2 but their thicknesses are 100 and 150 angstrom respectively.\n",
+    "We show the model that will be used graphically below. \n",
+    "\n",
+    "<center>\n",
+    "    <img src='sample/two_layers.png' alt='A slab model description of the two_layers system.' width='300px'></img>\n",
+    "</center>\n",
+    "<center>\n",
+    "    A slab model description of the two layer.\n",
+    "</center>\n",
+    "\n",
+    "To construct such a layer structure, first we create each of the materials, the associated layers, and the sub and super phases. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0f95d620-35b7-4b47-a3b4-9e33d5525b50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sld_4 = Material(sld=4.0, isld=0, name='Sld 4')\n",
+    "sld_4_layer = Layer(material=sld_4, thickness=100, roughness=2, name='SLD 4 Layer')\n",
+    "\n",
+    "sld_8 = Material(sld=8.0, isld=0, name='Sld 8')\n",
+    "sld_8_layer = Layer(material=sld_8, thickness=150, roughness=2, name='SLD 8 Layer')\n",
+    "\n",
+    "vacuum = Material(sld=0, isld=0, name='Vacuum')\n",
```

Comment:
One of these should have no roughness (or it should not matter at least), I think it is probably sensible to not set it in the tutorial, to make it clearer which interface is not used

Reply:
changed roughness to 0